sdk/core/azure_core_amqp/src/socks5.rs

```diff
@@ -0,0 +1,783 @@
+// Copyright (c) Microsoft Corporation. All Rights reserved
+// Licensed under the MIT license.
+
+//! SOCKS5 proxy support for Azure AMQP connections
```

Comment:
This is great module documentation, but given its not `pub` when imported in `src/lib/rs`, no one is going to see it. Necessary information might be better served in the crate `README.md`.